daml-lf/spec/value.rst

```diff
@@ -461,49 +457,44 @@ As of version 6, these fields are included:
 
   [A-Za-z0-9._:-]+
 
-If ``relative`` is unset or false, this is an absolute contract ID;
-otherwise, this is a relative contract ID.
-
 message List
 ^^^^^^^^^^^^
 
-*since version 6*
+(*since version 10*)
 
 A homogenous list of values.
 
-As of version 6, these fields are included:
+As of version 10, these fields are included:
 
 * repeated `message Value`_ elements
 
-Every member of ``elements`` must conform to the same type.
+.. note: *this section is non-normative*
+
+   Every member of ``elements`` must conform to the same type.
 
 message Optional
 ^^^^^^^^^^^^^^^^
 
-*since version 6*
+(*since version 10*)
 
 An optional value (equivalent to Scala's ``Option`` or Haskell's
 ``Maybe``).
 
-Only ``Optional`` may be used to encode a Value that conforms to an LF
```

Comment:
what’s wrong with that?

Reply:
I do not see the point of this statement. There is currently only one way to encode an `Optional`, I do not see why we cannot change that in the future.